src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -1086,24 +1090,29 @@ private class OffsetAndRule {
         //
         // private ctor
         //
+#if FEATURE_WIN32_REGISTRY 
         [System.Security.SecurityCritical]  // auto-generated
-        private TimeZoneInfo(Win32Native.TimeZoneInformation zone, Boolean dstDisabled) {
-            
-            if (String.IsNullOrEmpty(zone.StandardName)) {
+        private TimeZoneInfo(Win32Native.TimeZoneInformation zone, Boolean dstDisabled)
+        {
+
```

Comment:
nit: as long as you're changing the formatting here, you could delete this blank line